addons/api/mirage/factories/storage-policy.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import factory from '../generated/factories/storage-policy';
+import permissions from '../helpers/permissions';
+import { faker } from '@faker-js/faker';
+import generateId from '../helpers/id';
+
+export default factory.extend({
+  id: () => generateId('pst_'),
+
+  authorized_actions: () =>
+    permissions.authorizedActionsFor('storage-policy') || [
```

Comment:
Are there permissions for attaching storage policies to scopes?

Reply:
These are permissions in general for CRUDL operations on storage policies.

Reply:
Right, I was asking if there is also an additional permission we need to be able to attach these policies to a scope specifically?

Reply:
Interesting, do you know what permission the backend will check when calling `AttachStoragePolicy` on a scope then?

Reply:
checking with the backend team as I couldn't find this information anywhere, but we have a story for attaching storage policy and will be tracked with that [story](https://hashicorp.atlassian.net/browse/ICU-12079)